src/main/java/org/jnode/fs/FSAttribute.java

```diff
@@ -0,0 +1,7 @@
+package org.jnode.fs;
+
+public interface FSAttribute {
+    public String getName();
+    public String getValue();
```

Comment:
Based on other attribute types you may have found (maybe in the CentOS image you've created), are the values typically strings or binary values?
I'm not sure if there's an expectation for attribute values to be strings. Maybe if we know that certain attributes store values as strings we can parse them that way, but I don't think that the jnode-fs library needs to know about what possible attributes there are - that would possibly be handled in Nuix Engine.
... basically I think that `getValue` should be `public byte[] getValue()`.

Reply:
done